Jenkinsfile

```diff
@@ -81,58 +81,15 @@ pipeline {
         stage('Code Quality') {
           when {
             anyOf {
-              branch 'main'
+              branch 'release/struts-6-7-x'
             }
           }
           steps {
             withCredentials([string(credentialsId: 'asf-struts-sonarcloud', variable: 'SONARCLOUD_TOKEN')]) {
-              sh './mvnw -B -Pcoverage -DskipAssembly -Dsonar.login=${SONARCLOUD_TOKEN} verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar --no-transfer-progress'
+              sh './mvnw -B -Pcoverage -DskipAssembly -Dsonar.login=${SONARCLOUD_TOKEN} verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar'
             }
           }
         }
-        stage('Build Source & JavaDoc') {
```

Comment:
stage('Build Source & JavaDoc') removed because it's only needed for main?

Reply:
It was duplication, there is a dedicated section when using Java 8